Adding bloop breaks scalafmt

Hello,
Thanks for creating bloop. I love it. I tried adding bloop to our workflow at work. We also use scalafmt. After adding bloop, whenever I try running `scalafmt` I get this error.

Bloop Version:
```scala
addSbtPlugin("ch.epfl.scala" % "sbt-bloop" % "1.0.0-M9")
```
scalafmt Version:
```scala
addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "1.4.0")
```

```bash
[error] Caused by: java.lang.ClassNotFoundException: scala.meta.inputs.Input
[error] 	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
[error] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[error] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[error] 	at org.scalafmt.sbt.ScalafmtPlugin$.$anonfun$scalaConfig$2(Sca
```

FWIW, we don't use source dependencies.

--- a/frontend/src/main/scala/bloop/Project.scala
+++ b/frontend/src/main/scala/bloop/Project.scala
@@ -9,8 +9,7 @@ import _root_.monix.eval.Task
 import bloop.bsp.ProjectUris
 import bloop.config.{Config, ConfigDecoders}
 import bloop.engine.ExecutionContext
-import metaconfig.{Conf, Configured}
-import org.langmeta.inputs.Input
+import metaconfig.{Conf, Configured, Input}
 
 final case class Project(
     name: String,

--- a/project/Dependencies.scala
+++ b/project/Dependencies.scala
@@ -19,7 +19,7 @@ object Dependencies {
   val mavenScalaPluginVersion = "3.2.2"
   val ipcsocketVersion = "1.0.0"
   val monixVersion = "2.3.3"
-  val metaconfigVersion = "0.6.0"
+  val metaconfigVersion = "0.7.0"
   val circeVersion = "0.9.3"
 
   import sbt.librarymanagement.syntax.stringToOrganization

